ui/static/ui/js/manage_taxonomies.jsx

```diff
@@ -323,15 +322,34 @@ define('manage_taxonomies', ['react', 'lodash', 'jquery', 'utils', 'bootstrap'],
     },
     updateVocabularyType: function(e) {
       this.setState({vocabularyType: e.target.value});
-      e.target.isChecked = true;
     },
     updateMultiTerms: function(e) {
       this.setState({multiTerms: e.target.checked});
     },
     submitForm: function(e) {
-      var API_ROOT_VOCAB_URL = '/api/v1/repositories/' + this.props.repoSlug +
-        '/vocabularies/';
       e.preventDefault();
+      if (_.isEmpty(this.state.name)) {
```

Comment:
Now it seems that using an integer for a vocab title is an issue, but for a different reason. Is it the case that when the API gets a request for a vocabulary with an integer for a slug, it attempts to look it up by id?